frontend/src/components/ui/blockGenerator/BlockGenerator.jsx

```diff
@@ -97,21 +99,33 @@ const BlockPreview = ({id, src}) => {
   )
 }
 
-const BlockTitle = ({ name, id, color, openView, actions, src}) => {
+
+const BlockTitle = ({ name, id, color, openView, actions, src, blockEvents }) => {
+  const handleViewClick = () => {
+    if (!src) {
+      // Convert blockEvents to a string and display it in an alert
+      alert("Block Event: " + JSON.stringify(blockEvents, null, 2));
```

Comment:
What's going to be the content of `blockEvents`. When I talked to @plparent he seemed to say that value was entirely unused. Wouldn't it also be more consistent with the UI to use the modal instead of an alert?